.github/workflows/ci.yml

```diff
@@ -0,0 +1,19 @@
+# GitHub Actions - CI for Go to build & test.  See ci-go-cover.yml and linters.yml for code coverage and linters.
```

Comment:
Just a questions, are these new files does not need a Copyright header like cover.yml?

Reply:
This is stuff I've stolen from fxamacker/cbor, it didn't have a header in the original so I didn't dare adding anything.